MySQL: disable foreign key checks

laysauchoa opened the pull request:
Article migration:
https://help.aiven.io/en/articles/5580571-can-i-disable-foreign-key-checks

I've tested all the steps. However, the extra command in the end did not work for me. I could not reproduce.

Fixes: https://github.com/aiven/devportal/issues/657

ftisiot commented on docs/products/mysql/howto/disable-foreign-key-checks.rst:
```diff
@@ -0,0 +1,95 @@
+
+To disable the foreign key checks for the session, you give an additional parameter when you connect to your Aiven for MySQL using the ``mysqlsh``:
+
+.. code::shell
```
Not sure why but this code is not shown in the preview

ftisiot commented on docs/products/mysql/howto/disable-foreign-key-checks.rst:
```diff
@@ -0,0 +1,95 @@
+
+.. code:: shell
+
+    mysql> SHOW VARIABLES LIKE 'foreign_key_checks';
```
Can we only show the actual code people need to copy/paste?
```suggestion
    SHOW VARIABLES LIKE 'foreign_key_checks';
```

ftisiot reviewed (changes requested):
Good one Laysa!
The command works for me!

I left some comments to make this article more "devportal" style, also please check the second `mysql` command is not shown in the preview

laysauchoa commented on docs/products/mysql/howto/disable-foreign-key-checks.rst:
```diff
@@ -0,0 +1,95 @@
+
+  * - Variable
+    - Description
+  * - ``USER_HOST``
```
I can replace to HOST if it helps

laysauchoa commented on docs/products/mysql/howto/disable-foreign-key-checks.rst:
```diff
@@ -0,0 +1,95 @@
+Disable foreign key checks
+==========================
+
+It can be useful to know how to disable foreign key checks. For example, your migration to Aiven for MySQL may face foreign key violations which can result in an error, similar to::
```
Changed slightly, but seems good!

laysauchoa commented:
@ftisiot thanks for the review! addressed the comments. The last command did not work for me as I said on the PR description. So I was not sure if it was correct.

ftisiot commented:
@laysauchoa is this ready for review?

laysauchoa commented:
> @laysauchoa is this ready for review?

yes, I have added the last command in the last review, would be possible to check again? thank you

ftisiot reviewed (approved):
LGTM
